statsmodels/tsa/varma_process.py

```diff
@@ -464,11 +466,14 @@ class VarmaPoly(object):
 
 
     '''
+
+    nvars = deprecated_alias('nvars', 'neqs', '0.11.0')
+
     def __init__(self, ar, ma=None):
         self.ar = ar
         self.ma = ma
         nlags, nvarall, nvars = ar.shape
-        self.nlags, self.nvarall, self.nvars = nlags, nvarall, nvars
+        self.nlags, self.nvarall, self.neqs = nlags, nvarall, nvars
```

Comment:
Do we really make this change and deprecate this?

Reply:
This is mainly an example case chosen to be small, clear, and out of the way.  `VARResults` is a much more relevant case.

I'm not necessarily the person to ask; I think `varma_process` should be removed entirely and most of its functionality covered by `wold`.